tools/hypernode-vagrant-runner/tests/unit/hypernode_vagrant_runner/utils/test_run_local_command.py

```diff
@@ -36,6 +39,35 @@ def test_run_local_command_writes_output_to_stdout(self):
             self.check_output.return_value
         )
 
+    def test_run_local_command_also_writes_stderr_and_stdout_to_stdout_in_case_of_nonzero_exit_in_python3(self):
+        if is_python_3():
+            self.check_output.side_effect = CalledProcessError(
+                1, cmd='echo 1', output='stdout', stderr='stderr'
+            )
+
+            with self.assertRaises(CalledProcessError):
+                run_local_command(['echo', '1'])
+
+            expected_calls = [
+                call('stdout'), call('stderr')
+            ]
+            # For loop instead of assertEqual(s) or assertCountEqual(s) because
+            # that is both Python 2 and 3 compatible.
+            for expected_call in expected_calls:
+                self.assertIn(expected_call, self.write_output_to_stdout.mock_calls)
+
+    def test_run_local_command_also_writes_stdout_to_stdout_in_case_of_nonzero_exit_in_python2(self):
```

Comment:
Could be yeah, I'd also like the is_python3 better as a decorator like 'skip_if_python3' but for now this should be ok